content/en/docs/v3.4/get-started/etcd.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Get Started with etcd
```

Comment:
--> Get started

And this file should be renamed to `content/en/docs/v3.4/get-started.md`

Reply:
Moving to next/quickstart, so the new title will be _Quickstart_